src/components/Resources/ResourcePage.js

```diff
@@ -3,46 +3,110 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import axios from 'axios';
 import PersonalMenu from '../PersonalMenu';
-import { Grid, Breadcrumbs, Typography } from '@material-ui/core';
+import { Grid, Breadcrumbs, Typography, Chip } from '@material-ui/core';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+});
 
 function ResourcePage({ matchProps }) {
   const [resource, setResource] = useState({});
+  const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     axios
       .get('/api/v1/resources/' + matchProps.match.params.guid)
       .then(function(response) {
         // handle success
         setResource(response.data);
+        setLoading(false);
       })
       .catch(function(error) {
         // handle error
         console.log(error);
       });
-  }, [matchProps.match.params.id]);
+  }, [matchProps.match.params.guid]);
+
+  const classes = useStyles();
+
+  const {
+    title,
+    url,
+    author,
+    tags,
+    media_type: mediaType,
+    description,
+    user,
+    date_published: datePublished,
+    modified,
+    paid,
+  } = resource;
 
   return (
-    <>
-      <Grid container spacing={1}>
-        <Grid item lg={3}>
-          <PersonalMenu />
-        </Grid>
-        <Grid item lg={9}>
-          <Breadcrumbs
-            separator={<NavigateNextIcon fontSize="small" />}
-            aria-label="breadcrumb"
-          >
-            <Link color="inherit" to="/resources">
-              Resources
-            </Link>
-            <Typography color="textPrimary">{resource.title}</Typography>
-          </Breadcrumbs>
-          <h2>{resource.title}</h2>
-          <pre>{JSON.stringify(resource, 0, 2)}</pre>
-        </Grid>
+    <Grid container spacing={1}>
+      <Grid item lg={3}>
+        <PersonalMenu />
+      </Grid>
+      <Grid item lg={9}>
+        <Breadcrumbs
+          separator={<NavigateNextIcon fontSize="small" />}
+          aria-label="breadcrumb"
+        >
+          <Link color="inherit" to="/resources">
+            Resources
+          </Link>
+          <Typography color="textPrimary">{title}</Typography>
+        </Breadcrumbs>
+        {loading ? (
+          <p>Loading...</p>
+        ) : (
+          <>
+            <Typography variant="h2" gutterBottom>
+              {title}
+            </Typography>
+            <div className={classes.subtitle}>
+              <Typography variant="subtitle1" gutterBottom>
+                <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
Doesn't `<Link>` handle URLs outside of the app also?

Reply:
From what I can tell, MaterialUI focuses more on styling than on the function for `Links`: https://material-ui.com/components/links/

Reply:
I did just find this example, is this what you had in mind? https://stackoverflow.com/questions/38573892/i-try-to-make-my-material-ui-raisedbutton-link-to-an-external-url-without-succes

Reply:
I thought it did at one point, but it we type something like 

```<Link color="inherit" to="https://google.com">
            Resources
          </Link>``` 

... that renders out into https://localhost:3002/https://google.com :(

Reply:
8813690 now includes a fix for this using the <Link> component.

Reply:
Please feel free to critique what I just put up.